x-pack/plugins/maps/public/classes/styles/vector/properties/dynamic_style_property.tsx

```diff
@@ -313,11 +325,11 @@ export class DynamicStyleProperty<T>
     };
   }
 
-  formatField(value: string | number | undefined): string | number {
+  formatField(value: string | number | undefined | null): string | number {
```

Comment:
Since `formatField` signature changed to include `null`, Does `IStyleProperty` interface definition of `formatField` also need to be updated for this change? Should this use `RawValue`

Reply:
yes, agreed.